openedx/features/student_certificates/urls.py

```diff
@@ -7,4 +7,6 @@
 
 urlpatterns = [
     url(r"^certificates/$", views.student_certificates, name="certificates"),
+    url(r"^shared_certificates/(?P<certificate_uuid>[0-9a-f]{32})$", views.shared_student_certificate,
```

Comment:
@muak50 let's renamed this to `achievements`

Reply:
> @muak50 let's renamed this to `achievements`

actually keep it singular as `achievement`

Reply:
@emraan004 actually `achievements/uuid` is the complete url, So uuid is something which makes it singular.
We have copied this pattern from edX's url `certificates/uuid`
Please let us know is still need to make it singular.